src/components/Comment/Comment.tsx

```diff
@@ -172,14 +200,24 @@ export const Comment = ({
                 Reply
               </button>
               <button className={commentControlsButtonStyles}>Share</button>
-              <button className={commentControlsButtonStyles}>Pick</button>
+              {/* Only staff can pick, and they cannot pick thier own comment */}
+              {user &&
+                user.badge.some(e => e.name === "Staff") &&
+                user.userId !== scopedComment.userProfile.userId && (
+                  <button
+                    onClick={pick}
+                    className={commentControlsButtonStyles}
+                  >
+                    {scopedComment.isHighlighted ? "Unpick" : "Pick"}
```

Comment:
If this PR isn't adding the ability to unpick a pick we shouldn't add the UI for that action. Or if we do it should be made clear that the logic is pending to help future developers / prevent it being missed. Maybe use the text: 'TODO: Unpick'

Reply:
great catch, super easy change to implement both